src/french.js

```diff
@@ -1,68 +1,434 @@
 class French {
-
+  /*
+      word: verb in infinitif form -> can be divided in a stem and an ending
+        TODO: pronominal verbs: 'se' + infinitif
+        TODO: passive form: 'être' + past participle
+      info: {
+        mood: 'Indicatif'/'Subjonctif'/'Impératif'/'Conditionnel'/'Infinitif'/'Participe'
+        tense: 'present'/'past'/'future'/'simple past'/..., (!!!tenses different from english tenses)
+        person (subject): '1'/'2'/'3',
+          '1': Je, nous (I, we),
+          '2': tu, vous (you),
+          '3': he, she, they, it, undefined ('on', people, infinitif verb, object, 'ça', 'cela', ...)
+        singular: 'true/false',
+          if undefined subject: 'on' (english: we as a whole, it), people, infinitif verb, qui, tout, rien, object -> is considered singular -> true
+          ! 'on' can also be used instead of 'nous' (but this use is to be avoided) -> is considered plural -> false
+          true -> singular -> 's'
+          false -> plural -> 'p'
+        formal: 'true/false',
+          'vous' can be used to adress a singular person in a formal way -> if the subject is 's2' and formal, it is conjugated as 'p2', but the past participle stays singular
+        gender: 'female/male/unknown',
+          undefined subject: ils (english: they), nous (english: we) , vous (you, adressing a group of persons), the people -> is conjugated as male, when we don't know who 'they', 'nous', 'vous' represent (male or female)
+          unknown: only for 'on' with the meaning 'it'
+          ! 'on' and objects 'can' also have a gender !
+      }
+  */
   conjugate (word, info) {
-    // Format for info: {tense: 'present', formal: 'true/false', gender: 'female/male/undefined', singular: 'true/false/undefined', person: '1'/'2'/'3'}
-    //TODO define 3 verb group types
+    switch (info.mood) {
+      case 'Indicatif':
+        switch (info.tense) {
+          case 'Présent':
+            return this.doIndicatifPresent(word, info);
```

Comment:
don't put break after return statements.

http://stackoverflow.com/questions/6330114/do-you-need-break-in-switch-when-return-is-used